partner_resource_delivery_schedule/tests/test_partner_expected_date.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from datetime import datetime
+
+from freezegun import freeze_time
+
+from odoo.tests import tagged
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+from odoo.addons.sale.tests.common import TransactionCase
+
+
+@freeze_time("2023-12-18")
+@tagged("post_install", "-at_install")
+class TestPartnerExpectedDate(TransactionCase):
+    def setUp(self):
+        super(TestPartnerExpectedDate, self).setUp()
```

Comment:
@Shide Could you use classmethod instead ?

FYI, to help performances, it is now better to inherit from https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/tests/common.py#L19

Reply:
Sure, this was an error!

Reply:
I want to use SetUp instead setUpClass because I'm doing an assertion when creating the resource.calendar:
https://github.com/OCA/delivery-carrier/pull/747/files?diff=unified&w=0#diff-db3a6b820ce4edee36af473c6f5c0039a4c82c8af235241eab74c6f79ba9eb40R51-R55

`cls` doesn't have assert methods.